### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by christoff-1992**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (MelissaHoweEmis) [inline ReadMe.md]: dont like it

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
```
Comment (MelissaHoweEmis) [inline ReadMe.md]: test

File: ReadMe.md (hunk 1 above)
Comment (MelissaHoweEmis) [inline ReadMe.md]: !fixed

File: ReadMe.md (hunk 2 above)
Comment (MelissaHoweEmis) [inline ReadMe.md]: !resolve

File: ReadMe.md (hunk 1 above)
Comment (MelissaHoweEmis) [inline ReadMe.md]: !resolve

Comment (christoff-1992) [inline ReadMe.md]: !fixed

File: ReadMe.md (hunk 2 above)
Comment (christoff-1992) [inline ReadMe.md]: !fixed

**Pull request merged**